index.bs

```diff
@@ -915,6 +875,87 @@ spec: encrypted-media-draft; for: EME; urlPrefix: https://w3c.github.io/encrypte
         </p>
       </section>
 
+      <section>
+        <h4 algorithm>
+          Check a MIME Type
+        </h4>
+        <p>
+          The <dfn id="check-a-mime-type">check a MIME Type</dfn> algorithm takes {{DOMString}} |contentType|, string |media|, and either
+          a {{MediaDecodingType}} or {{MediaEncodingType}} |encodeOrDecodeType|. It
+          returns either failure, or a boolean that represents if the MIME type of |contentType| is [=supported by the user agent=]:
+        </p>
+        <ol>
+          <li>
+            Let |mimeType| be the result of running <a>parse a
+            MIME type</a> with |contentType|.
+          </li>
+          <li>
+            If the value of |mimeType| is <code>failure</code>,
+            return <code>failure</code>.
+          </li>
+          <li>
+            If |encodingorDecodingType| is {{MediaDecodingType/webrtc}} ({{MediaDecodingType}})
```

Comment:
```suggestion
            If |encodeOrDecodeType| is {{MediaDecodingType/webrtc}} ({{MediaDecodingType}})
```

Reply:
Thanks! I've just pushed a commit to fix this, but using `encodingOrDecodingType` throughout.